data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,1002 @@
+		conversation
+			`You are directed to an official landing pad on your return to Hai-home, and it is not long before you are introducing your entourage to Sayari. Alondo in particular seems excited to be here and seems to have a knack for charming Sayari with polite and insightful, if rather simple, questions throughout the diplomatic niceties.`
+			`	You get down to business when Sayari inquires how your journey went.`
+			choice
+				`	"I don't recall there being nearly so many pirates."`
+			`	Sayari frowns and exhales a long breath through her nose, "Yes. Well, it seems that the rumors making their way through your media have attracted every opportunist this side of the Deep."`
+			`	"Of course," she says, "some of the pirates were aware of us, and we were aware of them being aware of us. We had a quiet arrangement with a select few where we rewarded them for keeping the secret well contained within their own circles."`
+			`	Alondo looks surprised, but Mr. Yang and Ms. Adrianopolous both merely nod.`
+			`	"Extra security forces are being called in, but we've been busier than usual in Wah Ki with our northern brethren."`
+			`	Mr. Yang pulls out a datapad. "Well then, I'm afraid I may have some bad news. May I?"`
+			`	He gestures to a display and then takes control when indicated. "One of the things I brought with me was Republic Intelligence's estimate of the effectiveness of various information containment measures."`
+			`	A series of charts are thrown up on the screen. You're not an expert at data analysis, but highlighted percentages ranging from 4% to 39% do not look promising. The final graphic is a map of stages of what is labeled as 'artifact penetration'. "Artifact penetration is the point at which a government cannot reasonably attempt to pretend something is other than the truth, at least not while still holding to the claim of being a representative government," explains Mr. Yang.`
+			`	He checks the date. "Our most optimistic estimates suggest the Republic will be forced to acknowledge the existence of the Hai within three months, or we would lose so much credibility as to compromise electoral stability. The actual time limit is probably sooner."`
+			choice
+				`	"Would that be so bad?"`
+			`	"It is always bad to be forced into acknowledging a secret. If we act sooner, we may control the manner of that secret's release," says Yang. "Plus, while the Hai have let the trickle of humans that find the wormhole settle here peacefully, a massive influx is a problem nobody really wants to have to deal with."`
+			`	Sayari looks at you. "We will have to come up with a plan, and quickly, but for the moment may I ask a favor of the war hero in the room? Please meet up with the security forces converging on Allhome and help them route the pirates back to the other side of the wormhole. It will buy us some breathing room and options."`
+			`	It is so hard to say "no" to a Hai and their cute squirrelly features.`
```

Comment:
If we want to use this to juxtapose against something later, maybe have this be said by a different character (Alondo?) instead of thought by the player. As with some other comments, this may be particularly off if the player has already met and interacted with the Hai before all this.